peekingduck/pipeline/nodes/input/live.py

```diff
@@ -26,15 +27,19 @@ class Node(AbstractNode):
     def __init__(self, config: Dict[str, Any]) -> None:
         super().__init__(config, node_path=__name__)
 
-        resolution = config['resolution']
+        self.resize_info = config['resize']
         input_source = config['input_source']
         mirror_image = config['mirror_image']
```

Comment:
Possible to log the old resolution here (always) and log the resized resolution (if resized)? I find this useful in past deployments.

Reply:
Added portion to log info the two resolutions